### PR TITLE
Add documentation about swap enablement on kubevirtci

### PR DESCRIPTION
**Pull request by Barakmor1**
Signed-off-by: bmordeha <bmodeha@redhat.com>

### CODE REVIEW

**Review by enp0s3 — COMMENTED** [review]
/approve

File: KUBEVIRTCI_LOCAL_TESTING.md [hunk 1]
```diff
@@ -52,6 +52,20 @@ export KUBEVIRT_DEPLOY_PROMETHEUS_ALERTMANAGER=true
 export KUBEVIRT_DEPLOY_GRAFANA=true
 ```
 
+#### start cluster with swap enabled
+To enable prometheus, please also export the following variables before running `make cluster-up`:
```
Comment (iholder101) [inline KUBEVIRTCI_LOCAL_TESTING.md:56]:
```suggestion
To enable KSM (Kernel Samepage Merging), please also export the following variables before running `make cluster-up`:
```

Comment (Barakmor1) [reply KUBEVIRTCI_LOCAL_TESTING.md]: Done Thanks

Comment (iholder101) [reply KUBEVIRTCI_LOCAL_TESTING.md]: Ah sorry, I meant swap obviously :)

**Pull request merged**